integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java

```diff
@@ -1326,7 +1326,7 @@ public String testProjection() {
 
         PanacheQuery<CatProjectionBean> projectionDistinctQuery = Cat
                 // The spaces at the beginning are intentional
-                .find("   SELECT   disTINct  c.name, cast(null as string), SUM(c.weight) from Cat c where name = :name group by name  ",
+                .find("   SELECT   disTINct  c.name as foo, cast(null as string) as bar, SUM(c.weight) as baz from Cat c where name = :name group by name  ",
```

Comment:
This is suspicious, why this test suddenly requires a change?

Reply:
Yeah, this should keep working.